uvobot.rb

```diff
@@ -10,6 +10,22 @@ def send_message(text)
   Curl.post(ENV['UVOBOT_SLACK_WEBHOOK'], payload.to_json)
 end
 
+def check_issue_today
+  urlTodayCheck = 'https://www2.uvo.gov.sk/evestnik/-/vestnik/aktual'
+  todayH1 = Regexp.new(Time.now.strftime('/%Y - %-d.%-m.%Y '))
+  html = Curl.get(urlTodayCheck).body
+  doc = Nokogiri::HTML.parse(html)
+  if todayH1.match(doc.css('h1')[1].inner_html)
```

Comment:
tu by mohlo stacit nieco ako 

```ruby
identifier = Time.now.strftime('%Y - %d.%m.%Y ')
doc.css('h1')[1].inner_html.include?(identifier)
```

Vrati to true alebo false implicitne. Tu funkciu by som nazval `issue_ready?`, to je standard pre boolean funkcie.